### ISSUE TITLE
Test Errors in WordPress 4.6

### ISSUE THREAD

**Issue by jarednova** [timber/timber#1293]
### Expected behavior
All test pass

### Actual behavior
```
1) TestTimberImage::testThemeImageResize
copy(http://example.org/wp-content/themes/twentysixteen/images/cardinals.jpg): failed to open stream: HTTP wrapper does not support writeable connections
/home/travis/build/timber/timber/tests/test-timber-image.php:710
phar:///home/travis/.phpenv/versions/5.6.5/bin/phpunit/phpunit/TextUI/Command.php:152
phar:///home/travis/.phpenv/versions/5.6.5/bin/phpunit/phpunit/TextUI/Command.php:104

2) TestTimberImage::testThemeImageLetterbox
copy(/tmp/wordpress/wp-content/themes/twentysixteen/images/cardinals.jpg): failed to open stream: No such file or directory
/home/travis/build/timber/timber/tests/test-timber-image.php:731
phar:///home/travis/.phpenv/versions/5.6.5/bin/phpunit/phpunit/TextUI/Command.php:152
phar:///home/travis/.phpenv/versions/5.6.5/bin/phpunit/phpunit/TextUI/Command.php:104
```

**Comment by jarednova** [comment]
Current test failure is...
```
There were 2 errors:
1) TestTimberImage::testThemeImageResize
Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Undefined index: dirname").
/home/travis/build/timber/timber/tests/assets/image-test.twig:1
/home/travis/build/timber/timber/vendor/twig/twig/lib/Twig/Template.php:403
/home/travis/build/timber/timber/vendor/twig/twig/lib/Twig/Template.php:411
/home/travis/build/timber/timber/vendor/twig/twig/lib/Twig/Environment.php:362
/home/travis/build/timber/timber/lib/Loader.php:74
/home/travis/build/timber/timber/lib/Timber.php:286
/home/travis/build/timber/timber/tests/test-timber-image.php:723
phar:///home/travis/.phpenv/versions/5.6.5/bin/phpunit/phpunit/TextUI/Command.php:152
phar:///home/travis/.phpenv/versions/5.6.5/bin/phpunit/phpunit/TextUI/Command.php:104
Caused by
PHPUnit_Framework_Error_Notice: Undefined index: dirname
/home/travis/build/timber/timber/lib/ImageHelper.php:383
/home/travis/build/timber/timber/lib/ImageHelper.php:492
/home/travis/build/timber/timber/lib/ImageHelper.php:66
/home/travis/build/timber/timber/vendor/twig/twig/lib/Twig/Template.php:432
/home/travis/build/timber/timber/vendor/twig/twig/lib/Twig/Template.php:403
/home/travis/build/timber/timber/vendor/twig/twig/lib/Twig/Template.php:411
/home/travis/build/timber/timber/vendor/twig/twig/lib/Twig/Environment.php:362
/home/travis/build/timber/timber/lib/Loader.php:74
/home/travis/build/timber/timber/lib/Timber.php:286
/home/travis/build/timber/timber/tests/test-timber-image.php:723
phar:///home/travis/.phpenv/versions/5.6.5/bin/phpunit/phpunit/TextUI/Command.php:152
phar:///home/travis/.phpenv/versions/5.6.5/bin/phpunit/phpunit/TextUI/Command.php:104
2) TestTimberImage::testThemeImageLetterbox
Undefined index: dirname
/home/travis/build/timber/timber/lib/ImageHelper.php:383
/home/travis/build/timber/timber/lib/ImageHelper.php:492
/home/travis/build/timber/timber/lib/ImageHelper.php:153
/home/travis/build/timber/timber/tests/test-timber-image.php:740
phar:///home/travis/.phpenv/versions/5.6.5/bin/phpunit/phpunit/TextUI/Command.php:152
phar:///home/travis/.phpenv/versions/5.6.5/bin/phpunit/phpunit/TextUI/Command.php:104
ERRORS!
Tests: 619, Assertions: 1007, Errors: 2, Skipped: 9.
```

**Comment by jarednova** [comment]
Closed by #1317
